### ISSUE TITLE
[enhancement] Include changelog.txt in addon.xml (Krypton & Leia)

### ISSUE THREAD

**Issue by Mike-KL** [primaeval/script.tvguide.fullscreen#106]
@primeaval, with Krypton and Leia, changelog.txt can also be included within added addon.xml as news.

<news></news>  Reference http://kodi.wiki/view/Addon.xml#.3Cnews.3E

I took a quick look at latest version of iplayerwww I had installed as noted it already provided changelog information in addon.xml which is presented in Kodi Estuary as follows

i.e. access, Addon -> Information

![screenshot000](https://cloud.githubusercontent.com/assets/16574667/26754375/3165627e-487a-11e7-8b15-7cef56d0c4b8.png)

Then select within information window, Click here to see latest changes...

![screenshot001](https://cloud.githubusercontent.com/assets/16574667/26754376/316d5b6e-487a-11e7-82db-99093684e713.png)

**Comment by primaeval** [comment]
Thanks. Is there a way to automatically link the changelog to the news. I don't want to update both as I also have to do it in the repo.

**Comment by Mike-KL** [comment]
I don't think so, thought it useful to draw attention to change and fact that changelog.txt appears to be a depreciated feature, Krypton onwards. (i.e. I also note changelog.txt is not included anymore with the iplayerwww addon)

Searching for information on this topic, does not seem to be much detail

Changes to the skinning engine for Kodi Krypton
https://forum.kodi.tv/showthread.php?tid=250862&pid=2360647#pid2360647

Kodi 17 - No show changelog for addons
https://forum.kodi.tv/showthread.php?tid=283158&pid=2377274#pid2377274

Changelog vs News confusion
https://forum.kodi.tv/showthread.php?tid=303941&pid=2499634#pid2499634

**Comment by primaeval** [comment]
Thanks. I'll see if I can add the latest few changes in the news section next release.
There is much too much meddling in Kodi for hobby devs to keep up with. 
It is no wonder so many devs drop out.

**Comment by primaeval** [comment]
I added it to the latest version but it is ridiculous. 
I use the confluence skin in krypton and it is flashing transparent text all over the info dialog for the addon.
What a mess.

**Comment by Mike-KL** [comment]
@primaeval, It works cleanly as expected with the Estuary Skin

![screenshot000](https://cloud.githubusercontent.com/assets/16574667/26762392/a43b06a6-4941-11e7-9ebb-bb432aec876f.png)
![screenshot001](https://cloud.githubusercontent.com/assets/16574667/26762391/a439ab26-4941-11e7-9398-c58948c368fa.png)
